pnpm-lock.yaml

```diff
@@ -754,6 +751,8 @@ importers:
       '@wordpress/element': 4.1.0
       '@wordpress/i18n': 4.3.0
       '@wordpress/icons': 6.2.0
+      chart.js: ^3.5.1
```

Comment:
Have we added two different packages for chart JS? Do we know which is the right one?